### DISCUSSION TITLE
Welcome to quarto-cli Discussions!

### DISCUSSION THREAD

**Discussion by dragonstyle** [quarto-dev/quarto-cli#343] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by juliantao** [comment]
Just notice this, thanks @dragonstyle.

Hi everyone, I am Julian Tao, an associate professor in Civil Engineering (Bio-inspired Geotechncis) at Arizona State University.  Our group is in the process of implementing reproducible research and we are lucky to have found Quarto.

I use Quarto to:
1. Write papers
2. Make teaching and technical presentations
3. Document workflows
4. Create websites

In this community, I mainly file issues to report bugs or to request features. 

Many thanks to the Quarto team! You are doing a fantastic job. Documenting, publishing, and sharing technical content are so much easier because of your work!

**Comment by waqi786** [comment]
Hi everyone! I'm Waqar Ali, and I'm excited to be part of this community. I'm passionate about machine learning and AI 🤖, and I'm here to connect with like-minded individuals, learn from others, and share my own knowledge and ideas 💡. Looking forward to collaborating and contributing to discussions to help us all grow together 🌱. Let’s make this community strong and supportive 💪😊.
